### PR TITLE
optimized

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: plugins/meme.py [hunk 1]
```diff
@@ -14,51 +11,14 @@
...
+        await message.err("`reply to only media...`")
         return
-    if not os.path.isdir(Config.DOWN_PATH):
-        os.makedirs(Config.DOWN_PATH)
-    await message.edit("```Memifying...```")
-    dls = await message.client.download_media(
-        message=replied,
-        file_name=Config.DOWN_PATH)
-    dls_loc = os.path.join(Config.DOWN_PATH, os.path.basename(dls))
-    if replied.sticker and replied.sticker.file_name.endswith(".tgs"):
```
Comment (rking32) [inline plugins/meme.py:32]: now `.tgs` also supported with `@MemeAuto_bot` ?

Comment (Krishna-Singhal) [reply plugins/meme.py:32]: yes 😎

File: plugins/cbutton.py [hunk 2]
```diff
@@ -21,22 +21,40 @@ async def create_button(msg: Message):
     if Config.BOT_TOKEN is None:
         await msg.err("First Create a Bot via @Botfather to Create Buttons...")
         return
-    string = msg.input_or_reply_raw
+    string = msg.input_raw
+    replied = msg.reply_to_message
+    file_id, file_ref, message = None, None, None
+    if replied:
+        message = await msg.client.get_messages(msg.chat.id, replied.message_id)
```
Comment (rking32) [inline plugins/cbutton.py:28]: why extra request ? `replied == message` isn't it ?

**Comment by rking32** [comment]
you have removed so many things here. i don't know why you removed them.

File: plugins/meme.py [hunk 3]
```diff
@@ -14,51 +11,14 @@
...
-        if replied.video:
-            await message.edit("```Memifying this gay video...```")
-        else:
-            await message.edit("```What a Gif, Memifying...```")
-        file_2 = os.path.join(Config.DOWN_PATH, "meme.png")
-        await take_screen_shot(dls_loc, 0, file_2)
-        if not os.path.lexists(file_2):
-            await message.err("```Media not found...```", del_in=5)
-            return
-        meme_file = file_2
```
Comment (rking32) [inline plugins/meme.py:51]: this part should include ig

**Comment by rking32** [comment]
seems nice

**Pull request merged**